pyccel/ast/literals.py

```diff
@@ -7,7 +7,7 @@
 from pyccel.utilities.metaclasses import Singleton
 
 from .basic              import PyccelAstNode, Basic
-from .datatypes          import (NativeInteger, NativeBool, NativeFloat,
+from .datatypes          import (NativeGeneric, NativeInteger, NativeBool, NativeFloat,
                                   NativeComplex, NativeString, NativeFloat)
```

Comment:
This import imports `NativeFloat` twice. Would you mind fixing it

Reply:
https://github.com/pyccel/pyccel/pull/1151/commits/b7c2e0fed930288786083369c581105b53034e76